static/js/containers/pages/b2b/B2BPurchasePage.js

```diff
@@ -113,6 +105,7 @@ export class B2BPurchasePage extends React.Component<Props, State> {
     const params = new URLSearchParams(this.props.location.search)
     const contractNumber = params.get("contract_number")
     const discountCode = params.get("discount_code")
+    // TODO: avoid unnecessary .replace if product_id is a number?
```

Comment:
What does this refer to?

Reply:
Note I left myself & forgot to follow up on. Also fixed